src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
@MishimaHaruna Hmm, I meant for the code-styling, spaces between bitwise binary operators. And the as discussed with @Jedzkie on IRC, the statement you suggested is the same as `!(` or `() == 0`.